src/components/GraphicsPane.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useEffect, useRef, KeyboardEvent, MouseEvent } from "react";
+import { Box } from "@mui/material";
+import "./GraphicsPane.css";
+import Context from "@mui/lab/node_modules/@mui/base/TabsUnstyled/TabsContext";
+
+type GraphicsPaneProps = {
+  content: string;
+  onInterrupt: () => void;
+};
+
+const GraphicsPane = (props: GraphicsPaneProps) => {
```

Comment:
To get rid of the state-based communication, you might need to convert this into a React class (see PyEditor, ParsonsEditor, Challenge). 
React functional components are great due to their simplicity, but for your case a class (with a runCommand method) makes more sense.